docs/rule/table-groups.md

```diff
@@ -54,11 +54,29 @@ This rule **allows** the following:
 </table>
 ```
 
-If you have a component with one of the table groups specified as its `tagName`, you should disable the rule inline:
+If you have a component with one of the table groups specified as its `tagName`, or it has no tag and instead only has valid children, you can
+allow it with your configuration.
+
+```js
+'table-groups': {
+  'allowable-tbody-components': 'some-component-with-tag-name-tbody',
+}
+```
 
 ```hbs
 <table>
-  {{!-- template-lint-disable table-groups  --}}
-  {{some-component-with-tagName-tbody}}
+  {{some-component-with-tag-name-tbody}}
 </table>
 ```
+
+## Configuration
+
+One of these:
+
+* boolean - `true` - Enables the rule to be enforced with no extra allowed components as children of table components.
+* object[] - with the following keys:
+  * `allowed-caption-components` - string[] - components to treat as having the caption tag (using kebab-case names like `nested-scope/component-name`)
```

Comment:
What is the benefit of having 5 options (one of each table group) instead of a single string array `allowed-components` option? Hard for me to tell why it's necessary, and it seems like it just adds extra options for us to maintain.

Reply:
If we only have `allowed-components`, then we don't get the benefits of enforcing ordering. We wouldn't be able to tell that
```
<table>
  <MyTfoot />
  <MyThead />
</table>
```
is invalid.

Reply:
I see. The rule documentation doesn't mention anything about enforcing order so I didn't realize it did that. Can you update the doc to mention the ordering enforcement?